py/client-ticking/README.md

```diff
@@ -1,78 +1,95 @@
 # Deephaven Python Ticking Package
 
-The pydeephaven-ticking package helps you accesss Deephaven ticking data from Python. It extends
+The `pydeephaven-ticking` package enables access to ticking Deephaven data from Python. It extends
 the functionality of the pydeephaven package to add the ability to subscribe to tables and receive
-change notifications containing added, removed, and modified data. This package uses Cython to
-create a thin wrapper around the Deephaven native C++ library.
+change notifications containing added, removed, and modified data. It uses Cython to create a thin
+wrapper around Deephaven's C++ library.
 
 ## Prerequisites
 
-Before installing this library, you will need to install the Deephaven Core C++ library and the
-pydeephaven python package. All three packages (Deephaven Core C++, pydeephaven, Deephaven Core,
-and pydeephaven-ticking) are present in the Deephaven Core github repository. We assume you
-have checked out this repository at the location specified by `${DHROOT}`.
+`pydeephaven-ticking` requires a Linux operating system running on x86_64 architecture.
```

Comment:
This is not true. We only provide *pre-built* wheels for Linux x86_64, but determined individuals can build this for other platforms I'm sure.